docs/guides/operators.md

```diff
@@ -147,6 +147,15 @@ schema @server(responseValidation: true) {
 Disabling this setting will offer major performance improvements, but at the potential expense of data.
 :::
 
+### responseHeaders
+
+The `responseHeader` is an array of key-value pairs. These headers are added to the response of every request made to the server. This can be useful for adding headers like `Access-Control-Allow-Origin` to allow cross-origin requests, or some 
+additional headers like `X-Api-Key` to be used by the downstream services.
```

Comment:
Not sure if the x api key header is a good example.

Reply:
Should I replace it with something else or just remove it?

Reply:
@tusharmath I have updated the example.